nucliadb_node/nucliadb_node/settings.py

```diff
@@ -36,13 +37,7 @@ class Settings(BaseSettings):
 settings = Settings()
 
 
-class IndexingSettings(BaseSettings):
-    index_jetstream_target: Optional[str] = "node.{node}"
-    index_jetstream_group: Optional[str] = "node-{node}"
-    index_jetstream_stream: Optional[str] = "node"
-    index_jetstream_servers: List[str] = []
-    index_jetstream_auth: Optional[str] = None
-
+class IndexingSettings(utils_settings.IndexingSettings):
```

Comment:
With this change the dependency to utils is a bit bigger at the node.
If we are not planning on getting rid of it any time soon, then I think it's OK

Reply:
I thought any change would be moving to rust right? So for any python changes, the utils package is our tooling package to use for this sort of thing.

We need something like that for rust though probably